### PR TITLE
Improvement for README

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Add a status badge <br/> Improve Google doc link (remove unnecessary parameters)
| Fixed ticket |

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,6 +2,10 @@ Hello,
 
 This repository aims to improve the overall maturity of the PrestaShop open source project.
 
+It also hosts the source of the website https://www.prestashop-project.org/
```
Comment (PierreRambaud) [inline readme.md:5]:
```suggestion
It also hosts the source of the website [https://www.prestashop-project.org/](https://www.prestashop-project.org/)
```

**Review by atomiix — APPROVED** [review]

File: readme.md [hunk 2]
```diff
@@ -2,6 +2,10 @@ Hello,
 
 This repository aims to improve the overall maturity of the PrestaShop open source project.
 
+It also hosts the source of the website [https://www.prestashop-project.org/](https://www.prestashop-project.org/)
```
Comment (Progi1984) [inline readme.md:5]:
```suggestion
It also hosts the source of the website [https://www.prestashop-project.org](https://www.prestashop-project.org)
```

**Review by Progi1984 — COMMENTED** [review]
Small suggestion

**Comment by matks** [comment]
Closing just like I closed https://github.com/PrestaShop/open-source/pull/67

**Comment by kpodemski** [comment]
Hi @matks 

Would you like to resolve conflicts and improve readme? Your suggestions seem to be still relevant.

**Comment by matks** [comment]
@kpodemski 🎉 rebased 🚀

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thank you @matks :)

**Pull request merged**